docs/how-to/dummy-data.md

```diff
@@ -16,7 +16,7 @@ There are three ways to use dummy data in ehrQL.
 
 ehrQL is designed so that the same command can be used to output a dummy dataset when run on your own computer and then output a real dataset when run inside the secure environment as part of an OpenSAFELY pipeline.
 
-Refer to the [documentation on generating a dummy dataset](../../tutorial/generating-a-dummy-dataset/#generate-a-dummy-dataset) to see how this works.
+Refer to the [documentation on generating a dummy dataset](../tutorial/generating-a-dummy-dataset/index.md) to see how this works.
```

Comment:
thought: It seemed unnecessary linking to the section, because the section forms the entirety of the page.